strangerstudios/paid-memberships-pro
# Add zh_CN language translation

### All Submissions:

* [*] Have you followed the [Contributing guideline](https://github.com/strangerstudios/paid-memberships-pro/blob/master/.github/CONTRIBUTING.md)?
* [*] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [*] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/strangerstudios/paid-memberships-pro/pulls) for the same update/change?





### Changes proposed in this Pull Request:



This PR add the Simple Chinese support of zh_CN translation.

### How to test the changes in this Pull Request:

1. Add the .po and .mo files into languages/ directory;
2. Select the system languange to zh_CN;
3. Refresh the web page to check if it shows the Simple Chinese

### Other information:

* [*] Have you added an explanation of what your changes do and why you'd like us to include them?
* [*] Have you successfully run tests with your changes locally?



### Changelog entry

> Add zh_CN Simple Chinese language translation.
